src/EasyCaching.LiteDB/DefaultLiteDBCachingProvider.cs

```diff
@@ -321,6 +321,16 @@ public override void BaseSetAll<T>(IDictionary<string, T> values, TimeSpan expir
             return GetDict<T>(list);
         }
 
+        public override IDictionary<string, CacheValue<T>> BaseGetAll<T>()
+        {
+            throw new NotImplementedException();
```

Comment:
Is `NotSupportedException` better than `NotImplementedException` or not?

Reply:
Yes. It's better. I change it.